src/networks.ts

```diff
@@ -0,0 +1,119 @@
+type NetworkInfo = {
+  shortName: string;
+  chainID: number;
+  name: string;
+  currencySymbol: string;
+  baseAPI?: string;
+};
+
+export const networkInfo = new Map<number, NetworkInfo>([
+  [
+    1,
+    {
+      chainID: 1,
+      name: "Ethereum",
+      shortName: "eth",
+      currencySymbol: "ETH",
+      baseAPI: "https://safe-transaction.gnosis.io/api/v1",
+    },
+  ],
+  [
+    4,
+    {
+      chainID: 4,
+      name: "Rinkeby",
+      shortName: "rin",
+      currencySymbol: "RIN",
+      baseAPI: "https://safe-transaction.rinkeby.gnosis.io/api/v1",
+    },
+  ],
+  [
+    5,
+    {
+      chainID: 5,
+      name: "Goerli",
+      shortName: "gor",
+      currencySymbol: "GOR",
+      baseAPI: "https://safe-transaction.goerli.gnosis.io/api/v1",
+    },
+  ],
+  [
+    10,
+    {
+      chainID: 10,
+      name: "Optimism",
+      shortName: "oeth",
+      currencySymbol: "OETH",
+      baseAPI: "https://safe-transaction.optimism.gnosis.io/api/v1",
+    },
+  ],
+  [
+    56,
+    {
+      chainID: 56,
+      name: "Binance Smart Chain",
+      shortName: "bnb",
+      currencySymbol: "BNB",
+      baseAPI: "https://safe-transaction.bsc.gnosis.io/api/v1",
+    },
+  ],
+  [
+    100,
+    {
+      chainID: 100,
+      name: "Gnosis Chain (formerly xDai)",
+      shortName: "xdai", // gno this is the offical shortname. wierdly gnosis Safe still uses xdai
```

Comment:
Not sure I understand the comment here. Short name is that the short name for the chain or the currency?

Reply:
It's the shortname for the chain. It is used for the eip3770 address format i.e. `xdai:0x8e6c423585eC174306caF372132663c00Df877a3`.
According to the standard it should already be `gno:0x8e6c423585eC174306caF372132663c00Df877a3`. 

But I'll keep it like this for the moment.

Reply:
Works for me!